.github/issue_template.md

```diff
@@ -1,42 +0,0 @@
-<!-- NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue -->
```

Comment:
Thanks for this PR :)

Rather than removing this file, could you instead move it to the `.gitea` directory (which doesn't yet exist), so that when we move to gitea.com if issue templates don't yet exist as a feature in Gitea then we still have this as a fallback

Reply:
I can definitely do that.  I wasn't aware that there is a `.gitea` directory, and I will add the file accordingly.